src/NServiceBus.Persistence.AzureStorage.AcceptanceTests/ConfigureEndpointAzureStorageQueueTransport.cs

```diff
@@ -13,7 +13,7 @@ public class ConfigureEndpointAzureStorageQueueTransport : IConfigureEndpointTes
     {
         var connectionString = settings.Get<string>("Transport.ConnectionString");
         //connectionString = "UseDevelopmentStorage=true;";
-        configuration.UseSerialization<XmlSerializer>();
+        configuration.UseSerialization<JsonSerializer>();
```

Comment:
I see your point @timbussmann . Reminding that `Xml` is highly unlikely to be used in Azure scenarios.